source/wp-content/themes/wporg-news-2021/sass/blocks/_post-title.scss

```diff
@@ -1,3 +1,3 @@
 .wp-block-post-title {
-	margin-top: 0;
+	margin-top: 0 !important; // TODO this might be fixable via the blockGap prop in theme.json.
```

Comment:
yeah, i wonder if we should just set it to `0`? pretty much every element in the header and footer will need it manually turned off. 

or maybe we have a single css rule that disables it for the header, footer, and all their descendents? it could be useful for the content area, just not in the locked templates

https://github.com/WordPress/gutenberg/pull/33812

Reply:
this is interesting:

https://every-layout.dev/layouts/stack/

Reply:
> a single css rule

this seems to work so far:

```css
[class*="wp-container-"].site-header,
[class*="wp-container-"].site-footer,
[class*="wp-container-"].site-header > * + *,
[class*="wp-container-"].site-footer > * + *,
.site-header [class*="wp-container-"] > * + *,
.site-footer [class*="wp-container-"] > * + * {
    margin-top: 0;
}
```

...but now i have to update _every_ individual element inside the header/footer that is supposed to have a margin other than `0`, and make the selectors more specific. maybe there's a better way?

Reply:
As we run into problems with this, it'd be good to add them here:

https://github.com/WordPress/gutenberg/issues/34716

Reply:
Removed the `!important` here in 546e280